html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -241,6 +241,19 @@ has_field 'nbregpages' =>
     default => 0,
   );
 
+=head2 sms_pin_retry_limit
+
+=cut
+
+has_field 'sms_pin_retry_limit' =>
+  (
+    type => 'PosInteger',
+    label => 'SMS Pin Retry Limit',
+    default => 0,
+    tags => { after_element => \&help,
+             help => 'How many times a user can retry an sms pin having to request anothor pin. A value of 0 mean disables retries check.' },
```

Comment:
I would rephrase this: 

_Maximum number of times a user can retry a SMS PIN before having to request another PIN. A value of 0 disables the limit._